bin/validateSchema.ts

```diff
@@ -367,24 +394,23 @@ async function validateDeleteVisitsError429Schema() {
 }
 
 /**
- * Validates ErrorVisitsDelete400Response schema
+ * Validates ErrorVisitor400Response schema
  */
-async function validateErrorVisitsDelete400Schema(testSubscriptions: TestSubscription[]) {
-  console.log('\nValidating DeleteVisitsError400 schema: \n');
-  const deleteVisitsError400Schema = convertOpenApiToJsonSchema(
-    OPEN_API_SCHEMA,
-    '#/definitions/ErrorVisitsDelete400Response'
-  );
-  const deleteVisitsError400Validator = ajv.compile(deleteVisitsError400Schema);
+async function validateErrorVisitor400Response(testSubscriptions: TestSubscription[]) {
+  const schemaName = 'ErrorVisitor400Response';
+  console.log(`\nValidating ${schemaName} schema: \n`);
+  const visitorError400Schema = convertOpenApiToJsonSchema(OPEN_API_SCHEMA, '#/definitions/ErrorVisitor400Response');
+  const visitorError400Validator = ajv.compile(visitorError400Schema);
 
   // Validate against example file
-  ['./examples/delete_visits_400_error.json'].forEach((examplePath) =>
-    validateJson({
-      json: JSON.parse(fs.readFileSync(examplePath).toString()),
-      jsonName: examplePath,
-      validator: deleteVisitsError400Validator,
-      schemaName: 'DeleteVisitsError400',
-    })
+  ['./examples/delete_visits_400_error.json', './examples/shared/400_error_empty_visitor_id.json'].forEach(
```

Comment:
@JuroUhlar - I actually created an example for 400-incorrect-visitor-id in examples/shared/. We could use that. Let me make that change.

Reply:
I have now replaced all references to file `examples/delete_visits_400_error.json` with `examples/shared/400_error_incorrect_visitor_id.json`

Reply:
Good catch, thanks!